### ISSUE TITLE
Metered Pricing for Stripe Customer Portal

### ISSUE THREAD

**Issue by jbryanh** [stripe/stripe-firebase-extensions#163]
# Feature request

firestore-stripe-subscriptions
Add metered billing for this extension and Stripe Customer Portal

## Is your feature request related to a problem? Please describe.

A clear and concise description of what you want and what your use case is.

I charge per document from my firebase firestore.  I use flutter, and want to use the Stripe Customer Portal feature with the Stripe firestore subscriptions extension to charge metered billing.  Would love to have this available.  Everything else works great, just allowing that product category would be a huge step!

## Describe the solution you'd like

Everything is virtually there:
1. Provide any additional documentation required for developers to create the functions to write metered usage if any is necessary to work alongside the extension.  
2. Allow us to add metered products to the Customer Portal

## Describe alternatives you've considered

Only other option is to not use the Stripe Customer Portal and integrate payment and subscription info with in the web site.

**Comment by kylegraycar** [comment]
+1 to this. 

In case it helps, this is the error output: 
`❗️[Error]: Checkout session creation failed for doc [ymyWmrVdLUHBUyHFI9Kf]: Quantity should not be specified where usage_type is `metered`. Remove quantity from `line_items[0]``

**Comment by jbryanh** [comment]
@kylegraycar I am doing this now.  I think a subsequent update opened this up....or I just figured out how to do it.  I forgot about this request.  It's been a while since I've looked at it, though. I think remove the quantity, I see in my code I don't include it.  You're just adding the product, and you'll have to set up a cloud function to populate your usage, and then Stripe/Firebase will read that based on your configuration in Stripe Dashboard.
